org.eclipse.xtext.xbase.testlanguages/src-gen/org/eclipse/xtext/xbase/testlanguages/services/ContentAssistFragmentTestLangGrammarAccess.java

```diff
@@ -544,8 +544,8 @@ public ParserRule getXExpressionInClosureRule() {
 	}
 	
 	//XShortClosure XExpression:
-	//	=> ({XClosure} (declaredFormalParameters+=JvmFormalParameter (',' declaredFormalParameters+=JvmFormalParameter)*)?
-	//	explicitSyntax?='|') expression=XExpression;
+	//	=> ({XClosure} (declaredFormalParameters+=JvmFormalParameter (',' declaredFormalParameters+=JvmFormalParameter)*)
```

Comment:
Unexpected space after the predicate if followed by a parentheses.

Reply:
Will be fixed with next commit by allowing autowrap after question mark.